src/test/ui/consts/const-err2.rs

```diff
@@ -2,33 +2,34 @@
 // optimized compilation and unoptimized compilation and thus would
 // lead to different lints being emitted
 
+// revisions: default noopt opt opt_with_overflow_checks
```

Comment:
I may have gone overboard by having 4 revisions -- specifically, `default` (no extra flags) is likely a duplicate of one of the others, but I am not sure which. We already have *so many* tests, so I am somewhat inclined to remove `default` again... any opinions?

Reply:
Sure but there's such a thing as too much of any good thing. ;)